### PR TITLE
chore(docs/ExtractClientData): Update for mapextractor guideline

### PR DESCRIPTION
**Pull request by Si1ker**
## Description

Small changes to the guide so it can be easier for users

## Motivation and Context

Users are composed of a diverse type of people, the easier the guide is to read and follow the better, this can help non-techy people to use this tool appropriately.

### CODE REVIEW

File: docs/Extract-Client-Data.md [hunk 1]
```diff
@@ -15,26 +15,26 @@ Next press Configue and then Generate.
 ### Now that you have the source compiled, you can proceed to the next step.
 
 3. Go to your AzerothCore build directory (E.g. AzerothCore\build\bin\Release) and copy the following files to your World of Warcraft directory (E.g. D:\World of Warcraft 3.3.5a (no install))
-* **ace.dll**
 * **mapextractor.exe**
 * **mmaps_generator.exe**
 * **vmap4assembler.exe**
 * **vmap4extractor.exe**
-* **[extract_vmaps.bat](https://pastebin.com/KKMurZ4D)**
+* **[extract_vmaps.bat](https://pastebin.com/KKMurZ4D)** : You need to create a .txt file, write on it with the linked text, save it and then rename the file as **extract_vmaps.bat**
```
Comment (Yehonal) [inline docs/Extract-Client-Data.md:22]: uhm, actually it looks like a notepad window-based solution. You can safely create an extract_vmap.bat file and edit it with vscode. Ain't?

Comment (Kitzunu) [reply docs/Extract-Client-Data.md:22]: You can edit it with any kind of text editor. Notpad++, or even the default ones if you use "Open With" so it is not necessary no

Comment (Kitzunu) [reply docs/Extract-Client-Data.md:22]: But isnt the extract tools included with the build of the soruce?

File: docs/Extract-Client-Data.md [hunk 2]
```diff
@@ -15,26 +15,26 @@ Next press Configue and then Generate.
...
 
 4. First, you must extract **dbc** and **maps**. Go ahead and click on **mapextractor.exe**. You must wait until this process is completed.
 
 5. Next, we need to extract **Buildings**. Go ahead and click on **vmap4extractor.exe**. You must wait until this process is completed.
 
-6. Now we need to assembly **vmaps**. Click on **extract_vmaps.bat**. Same thing, you must wait until this process is completed.
+6. Now we need to assembly **vmaps**. Click on **extract_vmaps.bat**. You must wait until this process is completed and press a button when done.
```
Comment (Kitzunu) [inline docs/Extract-Client-Data.md:28]: Not really necessary to add this. But if you want to, what button? Any button?

File: docs/Extract-Client-Data.md [hunk 3]
```diff
@@ -15,26 +15,26 @@ Next press Configue and then Generate.
...
 
-7. Next, we need to extract **mmaps**. Go ahead and click on **mmaps_generator.exe**.  
+7. Next, we need to extract **mmaps**. In case you do not have the folder **mmaps** then create it manually. Go ahead and click on **mmaps_generator.exe**.
```
Comment (Kitzunu) [inline docs/Extract-Client-Data.md:30]: Wouldnt this apply to all folders?

File: docs/Extract-Client-Data.md [hunk 4]
```diff
@@ -15,26 +15,26 @@ Next press Configue and then Generate.
...
 
-6. Now we need to assembly **vmaps**. Click on **extract_vmaps.bat**. Same thing, you must wait until this process is completed.
+6. Now we need to assembly **vmaps**. Click on **extract_vmaps.bat**. You must wait until this process is completed and press a button when done.
 
-7. Next, we need to extract **mmaps**. Go ahead and click on **mmaps_generator.exe**.  
+7. Next, we need to extract **mmaps**. In case you do not have the folder **mmaps** then create it manually. Go ahead and click on **mmaps_generator.exe**.  
 **NOTE: This process will take up to a few hours, depending on your computer specs.**
 
 Now that everything is completed, you need to copy **dbc**, **maps**, **vmaps** and **mmaps** folders to your AzerothCore build directory (E.g. AzerothCore\build\bin\Release).
+Note: Keep in mind [this config line at worldserver.conf](https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/server/worldserver/worldserver.conf.dist#L61) when you place your dbc and maps into your build.
```
Comment (Kitzunu) [inline docs/Extract-Client-Data.md:34]:
```suggestion
Note: Keep in mind to change [this config line at worldserver.conf](https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/server/worldserver/worldserver.conf.dist#L61) when you place your dbc and maps into your build.
```

File: docs/Extract-Client-Data.md [hunk 5]
```diff
@@ -15,26 +15,26 @@ Next press Configue and then Generate.
...
 
-This guide was written by [mindsear](https://github.com/mindsear).
-
+Written by [mindsear](https://github.com/mindsear)
+Updated by [Silker](https://github.com/Si1ker)
```
Comment (Kitzunu) [inline docs/Extract-Client-Data.md:37]: Tbh, not to be "that guy" but this isn't necessary. It is all stored in the repository anyways

File: docs/Extract-Client-Data.md (hunk 1 above)
Comment (Si1ker) [reply docs/Extract-Client-Data.md:22]: not, that one, tha's why there's a link : )

File: docs/Extract-Client-Data.md (hunk 3 above)
Comment (Yehonal) [reply docs/Extract-Client-Data.md:30]: apparently only mmaps has this issue, I remember I had a similar situation

Comment (Kitzunu) [reply docs/Extract-Client-Data.md:30]: interesting

File: docs/Extract-Client-Data.md (hunk 2 above)
Comment (FrancescoBorzi) [reply docs/Extract-Client-Data.md:28]: I don't use Windows but I guess any button is fine

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request merged**